### PR TITLE
Support AbstractOperations with MakieExtension

### PR DESCRIPTION
**Pull request by glwagner**
Kinda cool to be able to do

```julia
heatmap(c^2)
```

and stuff

### CODE REVIEW

File: ext/OceananigansMakieExt.jl [hunk 1]
```diff
@@ -41,7 +66,7 @@ grids) with NaNs;
 - transferring data from GPU to CPU if necessary.
 """
 function make_plottable_array(f)
-
+    compute!(f)
```
Comment (navidcy) [inline ext/OceananigansMakieExt.jl:69]: so is `compute!(::Field) = nothing`?

Comment (glwagner) [reply ext/OceananigansMakieExt.jl:69]: yes https://github.com/CliMA/Oceananigans.jl/blob/840f6026c3025872e57c66e9b134ffbeb24bdc8c/src/Fields/field.jl#L440

**Review by navidcy — APPROVED** [review]
lgtm!

File: ext/OceananigansMakieExt.jl (hunk 1 above)
Comment (navidcy) [reply ext/OceananigansMakieExt.jl:69]: great! thanks! mostly my question was a sanity check for my understanding

**Pull request merged**